docs/sphinx/conf.py.in

```diff
@@ -344,6 +344,10 @@ texinfo_documents = [
 
 # -- Options for the linkcheck builder ----------------------------------------
 
+# Timeout value, in seconds, for the linkcheck builder
+if not (sys.version_info[0] == 2 and sys.version_info[1] <= 5):
```

Comment:
While it doesn't hurt, the minimum Python has been 2.6 for ages, so the check here is redundant.